qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uvcc_factory.py

```diff
@@ -72,84 +74,85 @@ def __init__(
+                These are: the evaluation count, the optimizer parameters for the
+                ansatz, the evaluated mean and the evaluated standard deviation.`
+            kwargs: any additional keyword arguments will be passed on to the VQE.
         """
-        self._quantum_instance = quantum_instance
-        self._optimizer = optimizer
-        self._initial_point = initial_point
-        self._gradient = gradient
-        self._expectation = expectation
-        self._include_custom = include_custom
-        self.ansatz = ansatz
-        self.initial_state = initial_state
         self._vqe = VQE(
-            ansatz=None,
-            quantum_instance=self._quantum_instance,
-            optimizer=self._optimizer,
-            initial_point=self._initial_point,
-            gradient=self._gradient,
-            expectation=self._expectation,
-            include_custom=self._include_custom,
+            **kwargs,
         )
+        self.quantum_instance = quantum_instance
+        self.optimizer = optimizer
+        self.initial_point = initial_point
+        self.gradient = gradient
+        self.expectation = expectation
+        self.include_custom = include_custom
+        self.ansatz = ansatz
+        self.initial_state = initial_state
+        self.callback = callback
 
     @property
     def quantum_instance(self) -> QuantumInstance:
         """Getter of the quantum instance."""
-        return self._quantum_instance
+        return self._vqe.quantum_instance
 
     @quantum_instance.setter
     def quantum_instance(self, q_instance: QuantumInstance) -> None:
         """Setter of the quantum instance."""
-        self._quantum_instance = q_instance
+        self._vqe.quantum_instance = q_instance
 
     @property
     def optimizer(self) -> Optional[Optimizer]:
         """Getter of the optimizer."""
-        return self._optimizer
+        return self._vqe.optimizer
 
     @optimizer.setter
     def optimizer(self, optimizer: Optional[Optimizer]) -> None:
         """Setter of the optimizer."""
-        self._optimizer = optimizer
+        self._vqe.optimizer = optimizer
 
     @property
     def initial_point(self) -> Optional[np.ndarray]:
         """Getter of the initial point."""
-        return self._initial_point
+        return self._vqe.initial_point
 
     @initial_point.setter
     def initial_point(self, initial_point: Optional[np.ndarray]) -> None:
         """Setter of the initial point."""
-        self._initial_point = initial_point
+        self._vqe.initial_point = initial_point
 
     @property
     def gradient(self) -> Optional[Union[GradientBase, Callable]]:
         """Getter of the gradient function"""
-        return self._gradient
+        return self._vqe.gradient
 
     @gradient.setter
     def gradient(self, gradient: Optional[Union[GradientBase, Callable]]) -> None:
         """Setter of the gradient function"""
-        self._gradient = gradient
+        self._vqe.gradient = gradient
```

Comment:
Similar comments in this class to the other factory so I am not going to repeat them. I commented here since this turns out to be different (like the expectation here too) and has no None test.